### ISSUE TITLE
Feedback for Xiaomi page

### ISSUE THREAD

**Issue by lockie** [urbandroid-team/dont-kill-my-app#159]
What about MIUI 11? Seems like this is current version, but the page lists instructions for MIUI 10

**Comment by AndroidDeveloperLB** [comment]
Pretty sure it has about the same issues.
Even worse, now it's officially violating the docs of Android 10 :

https://github.com/urbandroid-team/dont-kill-my-app/issues/161

**Comment by lockie** [comment]
Yeah, I'm sure it does. What about updating end user instructions for mitigating the issue?

**Comment by AndroidDeveloperLB** [comment]
I think those are the same too.
However, for developers, I think I've found this, which might help:
https://github.com/urbandroid-team/dont-kill-my-app/issues/162
Sadly I can't try it yet because I don't have a Xiaomi device.

**Comment by lockie** [comment]
> I think those are the same too.

Nope, they're not. Most of the system settings mentioned [here](https://dontkillmyapp.com/xiaomi#user-solution-title) are lacking in MIUI 11.

**Comment by AndroidDeveloperLB** [comment]
Sorry about that. I'm not familiar enough.
What was added, what was changed, and what was removed ?
